crates/store/re_data_loader/src/loader_lerobot.rs

```diff
@@ -528,7 +531,7 @@ fn load_scalar(
 }
 
 fn make_scalar_batch_entity_chunks(
-    field: &Field,
+    entity_path: String,
```

Comment:
strongly typed > stringly typed

```suggestion
    entity_path: EntityPath,
```